image.sparklyr.yaml

```diff
@@ -0,0 +1,39 @@
+name: "radanalyticsio/radanalytics-sparklyr"
+version: 1.0
+release: dev
+from: "rimolive/r-34-centos7:latest"
```

Comment:
for the moment, this is okay. But we should probably add the base image to docker.io/radanalyticsio